src/window.blp

```diff
@@ -8,101 +8,18 @@ Adw.ApplicationWindow documentation_viewer {
   height-request: 360;
   default-width: 1024;
   default-height: 768;
-  hide-on-close: true;
   title: bind webview.title;
 
   Adw.Breakpoint window_breakpoint {
     condition ("max-width: 850sp")
     setters {
       split_view.collapsed: true;
       button_sidebar.visible: true;
-      browse_page.width-request: 300;
-      search_page.width-request: 300;
```

Comment:
This is being done in code now

Reply:
``browse_page`` and ``search_page`` now called ``browse_view`` / ``search_view`` are not accessible from UI anymore so I have to do it in code